.openshift-ci/post_tests.py

```diff
@@ -204,6 +209,14 @@ def check_stackrox_logs(self):
             timeout=PostTestsConstants.CHECK_TIMEOUT,
         )
 
+    def add_test_output(self):
+        print("Storing operator test results in JUnit format")
+        self.run_with_best_effort(
+            ["scripts/ci/store-artifacts.sh", "store_test_results",
+             "operator/build/kuttl-test-artifacts"],
```

Comment:
I think based on @porridge changes there will be different dirs now for different tests. i.e. not all are "operator/build/kuttl-test-artifacts"

what about adding an array of junit test output passed to PostClusterTest() then this change is not so operator specific.